sdl-deltalake/src/main/scala/io/smartdatalake/workflow/dataobject/DeltaLakeTableDataObject.scala

```diff
@@ -85,6 +86,8 @@ import scala.util.Try
  * @param expectations List of [[Expectation]]s to enforce when writing to this data object. Expectations are checks based on aggregates over all rows of a dataset.
  * @param saveMode [[SDLSaveMode]] to use when writing files, default is "overwrite". Overwrite, Append and Merge are supported for now.
  * @param allowSchemaEvolution If set to true schema evolution will automatically occur when writing to this DataObject with different schema, otherwise SDL will stop with error.
+ * @param isManaged If set to true table location is managed by catalog, otherwise it is an external table and path must be defined.
```

Comment:
isManaged flag is not necessary. This property is defined by whenever the path property is set or not.

Reply:
Ok, removed